arras-energy/gridlabd-old
# Docker image clean up on install

This PR addresses issue(s) #117 and #149

## Current issues
None

## Code changes
utilities/docker/centos/gridlabd.sh 

## Test and Validation Notes
Before running the tests change the branch name (aivanova/origin_clean) and tag (test) in Makefile.conf 
1. Pull an image from Docker Hub and check the image size < 1GB. (Ensure GET_WEATHER option is set to no and REMOVE_SOURCE option is set to yes in Makefile.conf)
2. run `host% gridlabd --origin` to ensure that the files that should be ignored after install are not present.
